app/src/server/fineTuningProviders/openpipe/types.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
```

Comment:
I will remove all the models except our existing ones and `OpenPipe/mistral-ft-optimized-1215` in a future follow-up. They won't actually appear in the UI and I need them in the codebase for now because otherwise the analysis script I'm using to generate data for my blog post breaks.